af/fastpath/fastpath/core.py

```diff
@@ -350,52 +351,599 @@ def match_fingerprints(measurement):
     return matches
 
 
-@metrics.timer("score_measurement")
-def score_measurement(msm, matches):
-    """Calculate measurement scoring. Returns a summary dict
+def truekeys(d, keys):
+    """Check for values set to True in a dict"""
+    if isinstance(keys, str):
+        keys = (keys,)
+    for k in keys:
+        if d.get(k, None) != True:
+            return False
+
+    return True
+
+
+def falsekeys(d, keys):
+    """Check for values set to True in a dict"""
```

Comment:
`<bikeshed>`
Maybe these functions can have somewhere in them a the word `all`, maybe `all_keys_true`?

Initially I didn't understand it was looping over all of them and it's actually pretty similar to the python `all` construct.

Reply:
Good point, updated.